ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -618,7 +629,8 @@ func (r *RayClusterReconciler) reconcilePods(ctx context.Context, instance *rayv
 	if err := r.List(ctx, &headPods, common.RayClusterHeadPodsAssociationOptions(instance).ToListOptions()...); err != nil {
 		return err
 	}
-	if EnableBatchScheduler {
+	// this option is deprecated and will be removed soon
```

Comment:
I think the comment is meant for `EnableBatchScheduler`. Did you forget to remove it?

Reply:
Yeah, good catch.